{{cookiecutter.project_name}}/CONTRIBUTING.md

```diff
@@ -53,51 +51,52 @@ If you are proposing a feature:
 
 Ready to contribute? Here's how to set up **{{cookiecutter.project_name }}** for local development.
 
-1. Fork the **{{ cookiecutter.project_name }}** repo on {{ cookiecutter.vs | title }}.
-
-2. Clone your fork locally:
+1. Fork the **{{ cookiecutter.project_name }}** repository.
 
-        $ git clone ssh://git@{{ cookiecutter.vs_url }}{% if cookiecutter.vs|lower == "bitbucket" %}:7999/~{% else %}/{% endif %}your_name_here/{{ cookiecutter.project_name }}.git
+1. Clone your fork locally.
 
-3. Install your local copy into a virtualenv. Assuming you have virtualenv installed,
+1. Install your local copy into a virtualenv. Assuming you have virtualenv installed,
    this is how you set up your fork for local development:
 
         $ cd {{ cookiecutter.project_name }}/
-        $ virtualenv env
+        $ virtualenv --python=python3 env
```

Comment:
Unless Python2 might be supported in the future, perhaps it would be better to use `python3 -mvenv env`

Reply:
Problem with that approach is that `venv` is part of your Python installation and has a reduced set of functionality. So we could try it out and see whether there are some issues with that and then we could as well remove the dependency to virtualenv.

Since it is not a 1:1 change I would propose to try it out separately

Reply:
Ok. From a Linux perspective, it has been working for me (I don't have virtualenv installed). Maybe you can verify from your Mac?

Reply:
Open an issue to track this.

Reply:
Issue will be tracked here: https://github.com/tomtom-international/cookiecutter-python/issues/39